spec/requests/api/v1/docs/tags_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::Tags" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+
+  describe "GET /tags" do
+    before do
+      user.add_role(:admin)
```

Comment:
Should we set up the database with some followed tags like here:
https://github.com/forem/forem/blob/32548586efc9d365136a445c11836d19d18c1a44/spec/requests/api/v0/follows_spec.rb#L41-L44

Reply:
This is a great suggestion on point for how rswag works. With these models in the DB rswag will run the tests and display the example tags created by FactoryBot in the json file.

I'm totally cool with them being created in this `before` block or right under the [existing user up here](https://github.com/forem/forem/pull/18948/files#diff-0949f168a19ddabcc5bd6e34765476b92cc422d541041e155f669b29829f0f31R10) (with a `let!(:tags) { ...`) in that case. 

Another nitpick suggestion would be to use `create_list(:tag, 3)` 🙂